photon-api/src/main/scala/com/linkedin/photon/ml/estimators/GameEstimator.scala

```diff
@@ -614,6 +647,37 @@ class GameEstimator(val sc: SparkContext, implicit val logger: Logger) extends P
       }
   }
 
+  protected def prepareNormalizationContextWrappers(
```

Comment:
Why there are many empty lines in this function?

Reply:
This one is just an unfortunate function - regardless of what style convention is used, it looks like a mess